docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -146,7 +146,7 @@ Now you can select a hardware package that can cover these requirements. In this
 Camunda 8 defines four [cluster sizes](/components/concepts/clusters.md#cluster-size) you can select from (1x, 2x, 3x, and 4x) after you have chosen your [cluster type](/components/concepts/clusters.md#cluster-type). The following table gives you an indication of what requirements you can fulfill with each cluster size.
 
 :::note
-Contact your Customer Success Manager if you require a custom cluster size above these requirements.
+Contact your Customer Success Manager to increase the cluster size beyond the maximum 4x size. This requires custom sizing and pricing.
```

Comment:
the issue i am seeing here is that only new or renegiotiated contracts will actually get this - as the advanced-offering packages need to be unlocked for them first...

Reply:
@ultraschuppi I have added some text to the note to clarify this a bit more: 
![image](https://github.com/user-attachments/assets/23a057b1-4938-4796-88ec-3eb431e44bcf)